Dockerfile

```diff
@@ -0,0 +1,15 @@
+# build stage
+FROM golang:alpine AS build-env
+RUN apk --no-cache add build-base git bzr mercurial gcc
+RUN go get -u github.com/golang/dep/cmd/dep
+ENV D=/go/src/github.com/fnproject/fdk-testkit
+ADD . $D
```

Comment:
See my pull in fn core, should gopy only gopkg files, then do `dep ensure --vendor-only`, then do this add in a layer after that.

Reply:
addressed